src/reward.cpp

```diff
@@ -0,0 +1,60 @@
+#include "otpch.h"
+
+#include <iostream>
+#include "reward.h"
+
+
+Reward::Reward() :
+	Container(ITEM_REWARD_CONTAINER)
+{
+	maxSize = 32;
+	unlocked = true;
```

Comment:
When unlocked is set to false, the client renders a red line bordering the container slot when a player tries to move an item to that container. This is used for containers that are only for "ephemeral" storage, such as depot inbox and reward chests. It has nothing to do with `queryAdd`, though.